CHANGELOG.md

```diff
@@ -1,37 +1,99 @@
-# Changelog
-
 <!-- Instructions
 
-This changelog follows the patterns described here: <https://keepachangelog.com/en/1.0.0/>.
+This changelog follows the patterns described here: <https://keepachangelog.com/en/>.
 
 Subheadings to categorize changes are `added, changed, deprecated, removed, fixed, security`.
 
 -->
 
-## Unreleased
+# Changelog
+
+The latest published Vello release is [0.2.0](#020---2024-06-08) which was released on 2024-06-08.
+You can find its changes [documented below](#020---2024-06-08).
+
+## [Unreleased]
 
 ### Added
 
-- [#435](https://github.com/linebender/vello/pull/435) - Sweep gradients by [@dfrg](https://github.com/drfg)
-- [#538](https://github.com/linebender/vello/pull/538) - Impl `From<Encoding>` for `Scene` by [@waywardmonkeys](https://github.com/waywardmonkeys)
-- [#544](https://github.com/linebender/vello/pull/544) - Restore glyph hinting support by [@dfrg](https://github.com/drfg)
-- [#555](https://github.com/linebender/vello/pull/555) - Better glyph caching by [@dfrg](https://github.com/drfg)
-- [#563](https://github.com/linebender/vello/pull/563) - Use `vello_shaders` crate to load and preprocess WGSL by [@armansito](https://github.com/armansito)
+### Changed
+
+### Fixed
+
+## [0.2.0] - 2024-06-08
+
+### Added
+
+- Euler spiral based stroke expansion. ([#496] by [@raphlinus])
+- Sweep gradients. ([#435] by [@dfrg])
+- Bump allocation estimation. ([#436], [#454], [#522] by [@armansito])
+- Impl `From<Encoding>` for `Scene`. ([#538] by [@waywardmonkeys])
+- Glyph hinting support. ([#544] by [@dfrg])
+- Better glyph caching. ([#555] by [@dfrg])
+- `vello_shaders` crate to load and preprocess WGSL. ([#563] by [@armansito])
+- Coverage-mask specialization. ([#540] by [@armansito])
+- Support for the `#enable` post-process directive. ([#550] by [@armansito])
+- `wgpu-profiler` feature. ([#502] by [@DJMcNab])
```

Comment:
As far I can remember, this is localised to `with_winit`; not really something which has been "added" in a changelog sense

Is it worth making an `examples` header or similar?

Reply:
The feature also exists in `vello`, but apparently didn't work properly before, because you modified `vello` code in #502. So could move it under the fixed section, but because it didn't really work before, this is really the first release that has it.

As for an examples header, I'm leaning towards nope. In Druid we documented a bunch of stuff, including examples, in the changelog. However I think that leads to too much noise and so now I'm thinking that the changelog should focus more strictly on changes that directly impact an existing consumer.

Reply:
I believe it must have worked in 0.1.0 (see e.g. Xilem), but then regressed then unregressed

That is, I still don't think that this pr changed anything in the change from 0.1.0 to 0.2.0

Reply:
Alright, I'll remove it then.